runtime/mux_test.go

```diff
@@ -279,6 +325,17 @@ func TestMuxServeHTTP(t *testing.T) {
 	}
 }
 
+func unknownPathIs404(ctx context.Context, mux *runtime.ServeMux, m runtime.Marshaler, w http.ResponseWriter, r *http.Request, err error) {
+	if err == runtime.ErrUnknownURI {
+		w.WriteHeader(http.StatusNotFound)
+	} else {
+		stat, _ := status.FromError(err)
```

Comment:
You can use `status.Convert` for these cases. If we're not using a recent enough gRPC package, we should upgrade.

Reply:
Done. The version this repo uses (1.19.0) does indeed include that function.